File: src/main/java/pl/yalgrin/playnite/simplesync/repository/SeriesRepository.java
package pl.yalgrin.playnite.simplesync.repository;

import org.springframework.data.r2dbc.repository.Query;
import org.springframework.stereotype.Repository;
import pl.yalgrin.playnite.simplesync.domain.Series;
import reactor.core.publisher.Flux;

@Repository
public interface SeriesRepository extends ObjectRepository<Series> {
    @Query("select c.id from playnite_series c order by c.id asc")
    Flux<Long> findAllIds();
}

File: src/main/kotlin/pl/yalgrin/playnite/simplesync/library/repository/SeriesRepository.kt
package pl.yalgrin.playnite.simplesync.library.repository

import org.springframework.data.r2dbc.repository.Query
import org.springframework.stereotype.Repository
import pl.yalgrin.playnite.simplesync.library.domain.Series
import reactor.core.publisher.Flux

@Repository
interface SeriesRepository : ObjectRepository<Series> {
    @Query("select c.id from playnite_series c order by c.id")
    override fun findAllIds(): Flux<Long>
}
